x-pack/plugins/timelines/public/components/t_grid/styles.tsx

```diff
@@ -203,7 +203,7 @@ export const EventsTbody = styled.div.attrs(({ className = '' }) => ({
   className: `siemEventsTable__tbody ${className}`,
   role: 'rowgroup',
 }))`
-  overflow-x: hidden;
+  overflow-x: scroll;
```

Comment:
@XavierM Table is not responsive on small screen resolutions, but I guess this will be covered once moving to EUIDataGrid. So as a temporary solution I made the change directly to the timelines plugin to fix responsiveness. 

Is anybody using the t-grid component of timelines plugin at the moment?

Reply:
I'm not sure how this change might impact the current implementation without some thorough desk testing, so my inclination is to hold off on this change for now, and only revisit it if the transition to `EuiDataGrid` is temporarily delayed.

> Is anybody using the t-grid component of timelines plugin at the moment?

In addition to the o11y alerts page, the security solution is also using the t-grid component on several pages, behind a feature flag.

Reply:
@andrew-goldstein Ok I removed the horizontal scroll for now. Yep let's revisit responsiveness in case EuiDataGrid is delayed